cob_bringup/robots/cob4-2.xml

```diff
@@ -35,6 +35,10 @@
 				<arg name="robot" value="$(arg robot)"/>
 				<arg name="pc" value="$(arg cob4-2-b1)"/>
 			</include>
+                        <include file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```

Comment:
Copy :+1: Move :-1: 
If we move the launch file we will always need cob_robots (with thousands of deps) if we like to start the `stuck_detector`